### ISSUE TITLE
Add ELO for teams

### ISSUE THREAD

**Issue by mschmidt87** [jakobj/kicker_league#65]
Wie should add an ELO ranking for Teams as well. This includes:
* a ranking view for teams
* showing the team ELO in the EnterGame screen. 

For this, we should probably add a new teams database parallel to the players database.

**Comment by jakobj** [comment]
will we then show only the ELO from the team in the EnterGame screen? or also something like the individual ELOs? are we assuming the team elo is the "real" elo? then we could get rid of individual elos and calculate the single player elo from marginalizing the elos of all teams that player is part of. i am concerned that having two different rankings is not a good idea.

**Comment by mschmidt87** [comment]
Yes, that's a valid concern. I am not sure about the team ELOs anyway. We should keep in mind that with team ELOs, the statistic will be even more undersampled, thus worse. Especially if we might open the app for other people. Again, this could be tested with more samples.
